applications/staking_rewards.md

```diff
@@ -0,0 +1,129 @@
+
+Polkadot's staking ecosystem underpins its security, scalability, and governance. Selecting validators for the Decentralized Nodes Pool (DN Pool) requires balancing performance, decentralization, and fairness, yet current methods may lack transparency or fail to capture critical metrics. This project will establish a data-driven ranking methodology, supported by advanced tools and outreach, to empower both individual stakers and the network as a whole.
+
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Founders: Mirko Schmiedl, Jannik Schmiedl
+- Team: Staking Rewards is a trusted platform for staking insights, serving over 150,000 unique monthly users worldwide. With a focus on transparency and accessibility, the team has extensive experience in providing staking-related tools, analytics, and educational content. The team combines expertise in blockchain development, data science, and marketing to deliver impactful solutions.
+
+
+### Contact
+
+- **Contact Name:** Tobias Jung, Mirko Schmiedl
+- **Contact Email:** tobias@stakingrewards.com, mirko@stakingrewards.com 
+- **Website:** https://www.stakingrewards.com
+
+
+### Legal Structure
+
+- **Registered Address:** Schützengasse 7, 9000 St. Gallen, Switzerland
+
+- **Registered Legal Entity:** Finrate AG
+
+
+### Team's experience
+
+At StakingRewards.com, we are proud of our software development expertise, which has been key to building a platform that delivers accurate and comprehensive staking data. Our team specializes in creating complex systems that integrate seamlessly with a wide range of blockchain protocols, enabling real-time aggregation and analysis of staking rewards across networks. By focusing on clean, modular code and scalable architecture, we ensure that our platform remains robust and adaptable to the rapidly evolving blockchain ecosystem. Our efforts include developing intuitive user interfaces, APIs for institutional clients, and advanced data visualization tools, all designed to simplify the staking process. Backed by prominent investors like Galaxy Digital and CoinShares, we remain committed to advancing the platform’s capabilities and maintaining our position as a trusted resource for staking information.
+
+
+### Team Code Repos
+
+GitHub: https://github.com/stakingrewards
+Product: https://www.stakingrewards.com/
+
+
+### Team LinkedIn Profiles (if available)
+
+https://www.linkedin.com/company/stakingrewards/people/
+
+
+## Development Status :open_book:
+
+Currently Staking Rewards has developed a dedicated Asset Page featuring Polkadot Validators:
+https://www.stakingrewards.com/asset/polkadot
+
+
+## Development Roadmap :nut_and_bolt:
+
+Milestone 1 is the only Research Development Milestone. For future milestones, it needs to be evaluated if further research development is needed. Currently this is not planned.
+
+
+### Overview
+
+- **Total Estimated Duration:** 2 months
+- **Full-Time Equivalent (FTE):**  2 FTE
+- **Total Costs:** 15.000 Euro
```

Comment:
@Romao08 Thanks for confirming that USDC is fine. Can you please update total costs here to indicate USD?
```suggestion
- **Total Costs:** 15.000 USD
```